realm/realm-jni/src/io_realm_internal_table.cpp

```diff
@@ -1588,7 +1588,7 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_Table_nativeSetPrimaryKey(
     try {
         Table* table = TBL(nativeTablePtr);
         Table* pk_table = TBL(nativePrivateKeyTablePtr);
-        const char* table_name = table->get_name().data();
+        const char* table_name = table->get_name().substr(6).data(); // Remove "class_" prefix
```

Comment:
I believe we still uses Cores terminology in the internal API's, e.g. Table/Column/Rows.